src/components/principal/unit/resource/ResourceSummary.vue

```diff
@@ -46,6 +44,18 @@ export default {
       statusMenuPosition: "right"
     };
   },
+  computed: {
+    crewSize() {
+      let crewSize = 0;
+      if (this.resource.status.status == "IN_MISSION") {
```

Comment:
Consider using the value from contract. ( I remember I saw some enum on contract).
Also consider using ===